pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -153,7 +153,7 @@ func (c *Controller) processNextWorkItem() bool {
 }
 
 func (c *Controller) processPVC(pvcNamespace, pvcName string) error {
-	klog.V(4).Infof("Processing PVC %s/%s", pvcNamespace, pvcName)
+	klog.V(4).InfoS("Processing PVC", pvcNamespace, "/", pvcName)
```

Comment:
Should this be changed to use KRef or KObj? ditto everywhere

Reply:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging

![image](https://user-images.githubusercontent.com/9304834/106690061-084f7d80-660c-11eb-8e13-ae462507a02c.png)

I just follow the doc and fix.

Reply:
As part of the user guide at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#use-klogkobj-and-klogkref-for-kubernetes-objects, it also talks about using KObj and KRef

Reply:
Thanks, i know what you mean~

Reply:
> As part of the user guide at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#use-klogkobj-and-klogkref-for-kubernetes-objects, it also talks about using KObj and KRef

done~